pennylane/wires.py

```diff
@@ -545,15 +546,15 @@ def __ror__(self, other):
         return self.union(other)
 
     def intersection(self, other):
-        """Return the intersection of the current Wires object and either another Wires object or
-        an iterable that can be interpreted like a Wires object e.g., List.
+        """Return the intersection of the current :class:`~.Wires` object and either another :class:`~.Wires` object or
+        an iterable that can be interpreted like a :class:`~.Wires` object e.g., ``List``.
 
         Args:
-            other (Any): Wires or any iterable that can be interpreted like a Wires object
-                to perform the union with. See _process for details on the interpretation.
+            other (Any): :class:`~.Wires` or any iterable that can be interpreted like a :class:`~.Wires` object
+                to perform the union with. See ``_process`` for details on the interpretation.
```

Comment:
```suggestion
                to perform the intersection with. See ``_process`` for details on the interpretation.
```

Reply:
Same comment applies regarding ``_process``. The details could be either in `union` or `intersection`, and the other one can refer to that one, rather than to the private `_process`.